src/seedu/addressbook/data/AddressBook.java

```diff
@@ -20,15 +22,20 @@
  */
 public class AddressBook {
 
-    private final UniquePersonList allPersons;
+    private static final String ACTION_ADD = "ADD";
```

Comment:
good that you've used constants here to determine whether an add or delete is carried out